waiter/src/waiter/kv.clj

```diff
@@ -95,9 +95,9 @@
   given the underlying ZooKeeper implementation."
   [key]
   (when (re-matches #".*/.*" key)
-    (throw (ex-info "Key may not contain '/'" {:key key})))
+    (throw (ex-info "Key may not contain '/'" {:key key :log-level :warn})))
```

Comment:
Service names shouldn't reach this level.  Can we validate service names at the appropriate level?

Reply:
Yup. See my comment on the pullrequest. I was keeping this deliberately scoped narrow. We can keep it open to make that change.

Reply:
The trouble is that it should be considered an error to try to write an invalid key to ZK.  Let's make the change there or close this PR out.